app.py

```diff
@@ -0,0 +1,139 @@
+        
+        for file in commit['modified']: #Look at the files modified in each commit
+            split = file.split("/")
+            
+            if split[2] == 'vlans.yaml' : #Set flag to true if vlan change
+                vlanChange = True
+                
+            elif split[0] == 'host_vars' : #Add all changed switches to a list 
+                switchChanges.append(split[1]) #Adds switch name to list
+
+    print(appendTime("Vlan change: " + vlanChange + "\nSwitch chnages: " + switchChanges))
+
+    return(vlanChange, switchChanges)
+
+def runAnsible(vlanFlag, switchList): #Function for running ansible playbook
+    if vlanFlag: #If the vlanFlag is true run playbook on all switches else only on switches with changes
+        runAll = ansible_runner.run(
+            private_data_dir='ansible-switches/',
+            playbook='deploy.yaml'
+         )
+        print(appendTime("Vlan change, all switches update."))
+        return(runAll.status, runAll.stdout)
+
+    else:
+        strSwitchList = ','.join(map(str,switchList)) #Convert the list to string, items seperated by commas
+        runSwitches = ansible_runner.run(
+            private_data_dir='ansible-switches/',
+            playbook='deploy.yaml',
+            limit= strSwitchList #equivalent to --limit
+        )
+        print(appendTime("Individual switch change. " + strSwitchList + " switches update."))
+        return(runSwitches.status, runSwitches.stdout)
+    
+def sendAlert(vlanFlag, switchList, status, stdoutPath, webhook_url): #Function to send notification to slack channel
+
+    strSwitchList = ','.join(map(str,switchList)) #Convert the list to string, items seperated by commas
+
+    if status == 'failed':
+        color = '#f00216' #red color
+        title = ("Switch Change :large_red_square:")
+    else:
+        color = '#05eb2f' #green color
+        title = ("Switch Change :large_green_square:")
+
+    if vlanFlag: #Create the messages for the alerts, different depending on what switches were changed
+        message = 'Change made to switches.\nAll switches were changed.\nStatus:' + status + '\nPath to stdout: ' + str(stdoutPath.name)
+    else:
+        message = 'Change made to switches.\nSwitches: \n' + strSwitchList + '\nStatus: ' + status + '\nPath to stdout: ' + str(stdoutPath.name)
+
+    slack_data = {
+        "username": "SwitchAlert",
+        "icon_emoji": ":bulb:",
+        "channel": "#webhook-testing",
+        "attachments": [
+            {
+                "color": color,
+                "fields": [
+                    {
+                        "title": title,
+                        "value": message,
+                        "short": "false",
+                    }
+                ]
+            }
+        ]
+    }
+
+    headers = {
+        'Content-Type': "application/json",
+    }
+
+    response = requests.post(
+        webhook_url,
+        data=json.dumps(slack_data),
+        headers=headers
+    )
```

Comment:
Is it worth checking the status code of the response here and logging a message if there was an error?